docs/src/examples/sensitivity-analysis-svm.jl

```diff
@@ -94,59 +98,27 @@ Plots.plot!(p, svm_x, svm_y, label = "loss = $(round(loss, digits=2))", width=3)
 # - How does a change in labels of the data points (`y=1` to `y=-1`, and vice versa) affect the position of the hyperplane? This is achieved by finding the gradient of `w`, `b` with respect to `y[i]`, the classification label of the ith data point.
 # - How does a change in coordinates of the data points, `X`, affects the position of the hyperplane? This is achieved by finding gradient of `w`, `b` with respect to `X[i]`, 2D coordinates of the data points.
 
-# Note that finding the optimal SVM can be modelled as a conic optimization problem:
-
-# ```math
-# \begin{align*}
-# & \min_{x \in \mathbb{R}^n} & c^T x \\
-# & \text{s.t.}               & A x + s = b  \\
-# &                           & b \in \mathbb{R}^m  \\
-# &                           & s \in \mathcal{K}
-# \end{align*}
-# ```
-
-# where
-# ```math
-# \begin{align*}
-# c &= [l_1 - 1, l_2 -1, ... l_N -1, 0, 0, ... 0 \text{(D+1 times)}] \\\\
-
-# A &=
-# \begin{bmatrix}
-#  -l_1 &    0 & ... &    0 &            0 & ... & 0 & 0  \\
-#     0 & -l_2 & ... &    0 &            0 & ... & 0 & 0  \\
-#     : &    : & ... &    : &            0 & ... & 0 & 0  \\
-#     0 &    0 & ... & -l_N &            0 & ... & 0 & 0  \\
-#     0 &    0 & ... &    0 & -y_1 X_{1,1} & ... & -y_1 X_{1,N} & -y_1  \\
-#     0 &    0 & ... &    0 & -y_2 X_{2,1} & ... & -y_1 X_{2,N} & -y_2  \\
-#     : &    : & ... &    : &           :  & ... &          :   & :   \\
-#     0 &    0 & ... &    0 & -y_N X_{N,1} & ... & -y_N X_{N,N} & -y_N  \\
-# \end{bmatrix} \\\\
-
-# b &= [0, 0, ... 0 \text{(N times)}, l_1 - 1, l_2 -1, ... l_N -1] \\\\
-
-# \mathcal{K} &= \text{Set of Nonnegative cones}
-# \end{align*}
-# ```
-
-
 # ## Experiment 1: Gradient of hyperplane wrt the data point labels
 
 # Construct perturbations in data point labels `y` without changing the data point coordinates `X`.
 
-
-∇ = Float64[]
+∇ = zeros(N)
 dy = zeros(N);
 
 # begin differentiating
-for Xi in 1:N
-    dy[Xi] = 1.0  # set
+for i in 1:N
+    dy[i] = 1.0 # set
 
-    MOI.set(
-        model,
-        DiffOpt.ForwardInConstraint(),
-        cons,
-        MOI.Utilities.vectorize(dy .* index(b)),
-    )
+    for j in 1:N
+        MOI.set(
+            model,
+            DiffOpt.ForwardInConstraint(),
+            cons[j],
+            dy[j] * (dot(X[j,:], index.(w)) + index(b)),
+        )
+        # analogous to varying theta in the expression:
+        # (y_{i} + theta) (w^T X_{i} + b) \ge 1 - \xi_{i}
```

Comment:
there is a bug in Literate it seems where having a comment inside a loop will just split the code block and add text in the middle
```suggestion
```